src/to_do_components/todoList.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import TodoForm from './addToDo';
+
+class ToDoList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      todo:'',
+      newTodo:'',
+      todos: ['item 1', 'item 2'],
+    }
+  this.makeNewTodo = this.makeNewTodo.bind(this)
+  this.addTodo = this.addTodo.bind(this);
+  };
+
+  makeNewTodo(event) {
+    this.setState({newTodo: event.target.value});
+  }
+
+  addTodo() {
+    const todos = this.state.todos;
```

Comment:
This looks awesome. But you could simplify this to one line like `const { todos, todo, newTodo} = this.state`